TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/DriveTrain.java

```diff
@@ -31,52 +26,61 @@ public void runOpMode(){
         //telemetry resetter
         int counter = 0;
         //If disconnects during match - checks if encoder reset to 0
-        boolean encoderset = false;
+
+
         //If preset is active: Reset by manual movement, or when position reached.
         boolean presetactive = false;
         int mod = 1;
 
         waitForStart();
-
+        slide.reset();
         while (opModeIsActive()) {
             //   slowMode
             if (gamepad1.b) slowMode = !slowMode;
-            drive.move(gamepad1.left_stick_x * (slowMode ? 0.3 : .7), -gamepad1.left_stick_y * (slowMode ? 0.3 : .7), gamepad1.right_stick_x * (slowMode ? 0.3 : .7));
-            if (gamepad1.a) {slide.reset(); encoderset = true;}
+            drive.move(gamepad1.left_stick_x * (slowMode ? 0.3 : .6), -gamepad1.left_stick_y * (slowMode ? 0.3 : .6), gamepad1.right_stick_x * (slowMode ? 0.3 : .55));
 
 
             // Slides
             //Manual control, up
             if (gamepad1.left_trigger > 0) {
                 presetactive = false;
                 slide.upHold();
-                telemetry.addData("encoder postup", slide.getEncoder());
+                telemetry.addData("Up", slide.getEncoders()[0]);
                 telemetry.update();
                 ++counter;
             }
             //Manual control, down
             else if (gamepad1.right_trigger > 0) {
                 presetactive = false;
-                slide.downHold(encoderset);
-                telemetry.addData("encoder postdown", slide.getEncoder());
+                slide.downHold(true);
+                telemetry.addData("Down", slide.getEncoders()[0]);
                 telemetry.update();
                 ++counter;
-
             } else if (!presetactive) {slide.stop();}
 
-
             if (gamepad1.dpad_down) {slide.tozero(); presetactive = true;}
             if (gamepad1.dpad_left) {slide.low(); presetactive = true;}
             if(gamepad1.dpad_up) {slide.middle(); presetactive = true;}
             if (!slide.isBusy() && presetactive) {slide.stop(); presetactive = false;}
 
-
             //Claw
+            if (gamepad1.x){
```

Comment:
Why are there two sections for opening the claw?

gamepad1.x and gamepad1.left_bumper

Reply:
thats telemetry, renamed it